prow/cmd/deck/main.go

```diff
@@ -76,8 +77,15 @@ func main() {
 	}
 	ja.Start()
 
+	ta := &tideAgent{
+		log:  logrus.WithField("agent", "tide"),
+		path: *tideURL,
+	}
+	ta.start()
+
 	http.Handle("/", gziphandler.GzipHandler(http.FileServer(http.Dir("/static"))))
 	http.Handle("/data.js", gziphandler.GzipHandler(handleData(ja)))
+	http.Handle("/tide.js", gziphandler.GzipHandler(handleTide(ta)))
```

Comment:
the tideAgent (and this) should be wrapped in `if *tideURL != "" {`

Reply:
Done.